packages/contracts/test/manager/facets/ManagerFacet.t.sol

```diff
@@ -0,0 +1,203 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../DiamondTestSetup.sol";
+import "../../../src/dollar/interfaces/ICurveFactory.sol";
+import "../../../src/dollar/interfaces/IMetaPool.sol";
+import "../../../src/dollar/mocks/MockDollarToken.sol";
+import "../../../src/dollar/mocks/MockTWAPOracleDollar3pool.sol";
+
+import {GOVERNANCE_TOKEN_MINTER_ROLE, GOVERNANCE_TOKEN_BURNER_ROLE} from "../../../src/manager/libraries/LibAppStorage.sol";
+
+contract TestManagerFacet is DiamondSetup {
+    function testCanCallGeneralFunctions() public {
+        IManagerFacet.getExcessDollarsDistributor(contract1);
```

Comment:
This test isn't testing any assertions, it's only calling a function and showing that it won't revert.

Reply:
> This test isn't testing any assertions, it's only calling a function and showing that it won't revert.

This is just checking if the function exists in diamond after adding all facets to diamond

Reply:
That's not a useful or meaningful test

Reply:
> That's not a useful or meaningful test

No, it's not useful in managerFacet test, but useful in general diamond test